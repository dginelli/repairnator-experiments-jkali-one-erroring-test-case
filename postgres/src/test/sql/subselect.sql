select id from so_users where zdb('so_users', ctid) ==> '#subselect<id=<this.index>id>(id:[1,2,3,4,5,6,7,8,9,10])' order by id;
select id from so_users where zdb('so_users', ctid) ==> '#subselect<id=<so_posts.idxso_posts>id>(id:[1,2,3,4,5,6,7,8,9,10])' order by id;