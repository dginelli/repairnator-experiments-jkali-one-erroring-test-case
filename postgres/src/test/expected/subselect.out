select id from so_users where zdb('so_users', ctid) ==> '#subselect<id=<this.index>id>(id:[1,2,3,4,5,6,7,8,9,10])' order by id;
 id 
----
  2
  3
  4
  5
  6
  7
  9
(7 rows)

select id from so_users where zdb('so_users', ctid) ==> '#subselect<id=<so_posts.idxso_posts>id>(id:[1,2,3,4,5,6,7,8,9,10])' order by id;
 id 
----
  3
  4
  7
  9
(4 rows)

